docs/static/releasenotes.asciidoc

```diff
@@ -80,6 +81,18 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
 
+[[logstash-7-17-21]]
+=== Logstash 7.17.21 Release Notes
+
+No user-facing changes in Logstash core.
+
+==== Plugins
+
+*Multiline Codec - 3.1.2*
+
+* Fix: a race condition in periodic runner that blocks clean up process and pipeline shutdown https://github.com/logstash-plugins/logstash-codec-multiline/pull/72[#72]
```

Comment:
```suggestion
* Fixed a race condition in periodic runner that blocks clean up process and pipeline shutdown https://github.com/logstash-plugins/logstash-codec-multiline/pull/72[#72]
```

Reply:
There are a lot of examples of how to do these, but this tends to be the most common.